src/Fable.React.Helpers.fs

```diff
@@ -202,8 +202,9 @@ module Helpers =
         false
 #endif
 
-    /// Same as the default `React.memo` equality but ignores functions.
-    /// Useful in combination with `FunctionComponent.Of` `memoizeWith` parameter when F# equality isn't required
+    /// Memberwise referential comparison (same as default React.memo) that ignores
```

Comment:
I agree the comment is not very clear but the tricky part is this is not exactly referential comparison (for numbers and strings for example the comparison is by value). There's some [discussion about it here](https://github.com/fable-compiler/fable-react/pull/153#discussion_r276569792).

Reply:
Better now? Note that I use the term "value types", but I'm not sure exactly how the equality works with structs (e.g. `DateTime`) in the JS environment. Are they compared by value?